### PR TITLE
Adding vaex-hdf5 (Generated by https://github.com/maartenbreddels/releash)

### PR DESCRIPTION
**Pull request by maartenbreddels**

### CODE REVIEW

File: recipes/vaex-hdf5/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    - h5py
+
+test:
+  imports:
+    - vaex.hdf5
+
+about:
+  home: https://www.github.com/maartenbreddels/vaex
+  license: MIT
+  license_family: MIT
```
Comment (jakirkham) [inline recipes/vaex-hdf5/meta.yaml]: Is there a license file that we can include?

Comment (maartenbreddels) [reply recipes/vaex-hdf5/meta.yaml]: 👍 
Now needs to wait for [vaex-core](https://github.com/conda-forge/staged-recipes/pull/4083) to be merged

**Comment by maartenbreddels** [comment]
@jakirkham not sure if you get notifications when the build succeeds, in any case, *ping*

File: recipes/vaex-hdf5/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    - python
+    - vaex-core >=0.1
+    - h5py
+
+test:
+  imports:
+    - vaex.hdf5
+
+about:
+  home: https://www.github.com/maartenbreddels/vaex
```
Comment (dougalsutherland) [inline recipes/vaex-hdf5/meta.yaml]: Where does the source for this actually live? Not seeing it in that repo....

**Comment by dougalsutherland** [comment]
I think this can be `noarch: python`.

File: recipes/vaex-hdf5/meta.yaml (hunk 2 above)
Comment (maartenbreddels) [reply recipes/vaex-hdf5/meta.yaml]: it's a branch (packages), and there it lives in the packages directory. If this goes onto conda-forge  and it's all working (and some other package coming) i'm merging it to master).

**Comment by maartenbreddels** [comment]
Ah didn't think of that! I will do that.

File: recipes/vaex-hdf5/meta.yaml (hunk 2 above)
Comment (dougalsutherland) [reply recipes/vaex-hdf5/meta.yaml]: Ah, cool. :+1:

**Comment by jakirkham** [comment]
Just to make sure, this is pure Python?

**Comment by dougalsutherland** [comment]
Yes, it is: [source here](https://github.com/maartenbreddels/vaex/tree/packages/packages/vaex-hdf5), it's just a few Python files and an `h5py` dependency.

The one slightly weird thing is that both it and `vaex-core` install to `site-packages/vaex/`. Do we know if conda does anything weird about that, especially with `noarch: python`?

**Comment by maartenbreddels** [comment]
I've done test with pip, setuptools, and this seems to work fine, I hope conda/conda-forge will accept this. It is an alternative way to namespaces, that have restrictions.

**Comment by jakirkham** [comment]
Nothing weird that I know of. We do the same with the `backports.*` packages. Though we now provide `backports` to initialize the namespace. Would recommend doing something similar with this collection.

**Comment by maartenbreddels** [comment]
vaex-core basically does that, the difference with namespaces is that I do `vaex/__init__.py` only in vaex-core, the rest of the 'sub' packages don't have the `__init__.py`, so it's not really a namespace package. And vaex-core a dependency, so the `site-packages/vaex/` dir should always exist (if installed in the right order).

**Comment by maartenbreddels** [comment]
To clarify, afaik you can't have something meaningfull in your __init__.py, and they have to be same content, or it has to be empty, plus a number of language version issues, I thought this was a nice workaround.

**Comment by jakirkham** [comment]
Sounds like you are doing the same thing. Agree that it should be fine. :)

**Comment by maartenbreddels** [comment]
I'm wondering why a `noarch: python` builds on all platforms btw, or is that because the initial PR didn't include it?
Any case, @jakirkham || @dougalsutherland hope this is good to merge.

**Comment by dougalsutherland** [comment]
It runs everywhere on `staged-recipes`, which is good because it tests everywhere, but then the feedstock will only run on CircleCI.

This looks good to me now; merging. Thanks @maartenbreddels!

**Pull request merged**